ctapipe/tools/process.py

```diff
@@ -63,16 +64,36 @@ class ProcessorTool(Tool):
         default_value=False,
     ).tag(config=True)
 
+    energy_regressor_path = Path(
+        default_value=None,
+        allow_none=True,
+        exists=True,
+        directory_ok=False,
+        help="Path to a trained energy regression model (see ctapipe-ml-train-energy-regressor)",
+    ).tag(config=True)
+
+    particle_classifier_path = Path(
+        default_value=None,
+        allow_none=True,
+        exists=True,
+        directory_ok=False,
+        help="Path to a trained particle id classifier model (see ctapipe-ml-train-particle-classifier)",
+    ).tag(config=True)
+
     force_recompute_dl2 = Bool(
         help="Enforce dl2 recomputation even if already present in the input file",
         default_value=False,
     ).tag(config=True)
 
+    stereo_combiner_configs = List(Dict()).tag(config=True)
+
     aliases = {
         ("i", "input"): "EventSource.input_url",
         ("o", "output"): "DataWriter.output_path",
         ("t", "allowed-tels"): "EventSource.allowed_tels",
         ("m", "max-events"): "EventSource.max_events",
+        ("e", "energy-regressor"): "ProcessorTool.energy_regressor_path",
+        ("particle-classifier"): "ProcessorTool.particle_classifier_path",
```

Comment:
See above. No need for a short form for the energy regressor, but no short form for the particle classifier (`-p` is occupied)